hooks/useComputedValues.ts

```
@@ -0,0 +1,172 @@
+  structure: Record<string, any>,
+  onChange: (path: string, value: number) => void
+) {
+  const [computedValues, setComputedValues] = useState<ComputedValue[]>([]);
+  const previousDataRef = useRef<Record<string, any>>(data);
+
+  useEffect(() => {
+    const computed: ComputedValue[] = [];
+
+    function findComputedFields(
+      obj: Record<string, any>,
+      currentPath: string = ""
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)